test/integration/tlsroute_test.go

```diff
@@ -204,7 +206,14 @@ func TestTLSRoutePassthroughReferenceGrant(t *testing.T) {
 	cleaner.Add(service)
 
 	t.Logf("exposing deployment %s/%s via service", deployment2.Namespace, deployment2.Name)
+	const service2Port = 8443 // Use a different port that listening on the Gateway for TLS.
```

Comment:
Can you be more descriptive in here? This is the type of tribal knowledge that we're very good at accruing over time 🙃 Maybe refer to a file/line which defined that "default"?

+ 

```suggestion
	const service2Port = 8443 // Use a different port than Gateway's TLS listener port.
```